api/handler/acl.go

```diff
@@ -56,13 +56,17 @@ const (
 	s3GetObjectVersion           = "s3:GetObjectVersion"
 )
 
-// AWSACL is aws permission constants.
-type AWSACL string
+// enum of Amazon S3 ACL permissions.
+type amazonS3Permission string
 
+// various access permissions defined in
+// https://docs.aws.amazon.com/AmazonS3/latest/API/API_Grant.html.
+//
+// See also https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#permissions.
 const (
-	aclFullControl AWSACL = "FULL_CONTROL"
-	aclWrite       AWSACL = "WRITE"
-	aclRead        AWSACL = "READ"
```

Comment:
Especially these renames.

Reply:
I can't understand what `acl` stands for - Amazon S3 or NeoFS.